src/main/java/com/parasoft/findings/jenkins/coverage/api/metrics/steps/CoverageTableModel.java

```diff
@@ -110,12 +110,9 @@ public List<TableColumn> getColumns() {
                 .build();
         columns.add(packageName);
 
-        configureValueColumn("lineCoverage", Metric.LINE, Messages.Column_LineCoverage(),
-                Messages.Column_DeltaLineCoverage("Δ"), columns);
```

Comment:
Remove unused methods in Messages class and messaged, like Column_DeltaXXXCoverage(xx).